src/controller/python/build-chip-wheel.py

```diff
@@ -88,6 +96,34 @@ def __init__(self, name):
     # for many of its features.)
     os.chdir(tmpDir)
 
+    # Get current commit id
+    if repoDir != "":
+        try:
+            commitId = subprocess.check_output(['git', 'rev-parse', 'HEAD'], cwd=repoDir).decode("utf-8").strip()
+            branch = subprocess.check_output(['git', 'branch', '--show-current'], cwd=repoDir).decode("utf-8").strip()
+            gitDescribe = subprocess.check_output(['git', 'describe', '--dirty', '--always'], cwd=repoDir).decode("utf-8").strip()
+        except subprocess.CalledProcessError:
+            # We failed to get the commit if and branch of this CHIP checkout, maybe this checkout is not a git repo.
+            # Leaving commit id and branch empty.
+            pass
```

Comment:
Log instead of pass? Even an stdout message would help here

Reply:
I used pass since build with gn will not show stdout / stderr output by default if the command returns with 0. Added a output to stderr.